### ISSUE TITLE
Missing RPi section on emonSD build using EmonScripts

### ISSUE THREAD

**Issue by TrystanLea** [openenergymonitor/EmonScripts#25]
Missing RPi section on admin page on emonSD build using EmonScripts

**Comment by alexandrecuer** [comment]
is this what you mean by RPI section of the admin module ? If yes I do have it with an SD built using the Scripts.....
![RPI_section](https://user-images.githubusercontent.com/24553739/64685093-a32f4100-d486-11e9-8413-82759f15a2e2.png)

**Comment by TrystanLea** [comment]
Yes thats what I mean, thanks for the pic, its missing on RPI4 it seams..
